### DISCUSSION TITLE
V2样式如何兼容移动端,是否支持Ant Design Mobile

### DISCUSSION THREAD

**Discussion by huaxiangsirenbutong** [alibaba/formily#2338] [Q&A 2.x]
请问V2样式如何兼容移动端，如果不支持的话，有啥好的解决方案不

**Comment by cncolder** [comment]
我不是内部人员不了解 formily 未来的走向.

formily 默认配置适合后台表单的粗加工, 用在移动端精细 UI 这种场景你需要做的工作会很多. 另外 formily 是开放的, 组件可以不用 antd, 自己决定 schema 中的 node 用哪些组件渲染.
